client/components/inline-support-link/index.jsx

```diff
@@ -80,8 +86,9 @@ class InlineSupportLink extends Component {
 				rel="noopener noreferrer"
 				{ ...externalLinkProps }
 			>
-				{ showText && ( text || translate( 'Learn more' ) ) }
-				{ supportPostId && showIcon && <Gridicon icon="help-outline" size={ iconSize } /> }
+				{ shouldDisplayChildren && children }
+				{ shouldDisplayTextPropValue && ( text || translate( 'Learn more' ) ) }
```

Comment:
As a follow up Janitorial, we can maybe also simplify this to updating existing usages by removing the text prop/passing through text via children.

Reply:
Yeah, I think that'd be a good idea in the long run.